src/components/button/button.scss

```diff
@@ -8,6 +8,7 @@
   font-size: 15px;
   box-sizing: border-box;
   border: 1px solid transparent;
+  white-space: nowrap;
```

Comment:
Would the ellipsis mixin be something we want here instead?

Reply:
This just prevents the button from breaking into two lines, so I don't have to specify a width. But if we want to have ellipses for buttons, I can go back and add a width.

Reply:
I think we're okay without for now